metadb/server.go

```diff
@@ -230,3 +239,24 @@ func (s *service) startServer(ctx context.Context) error {
 	log.WithContext(ctx).Info("metadb service is stopped")
 	return nil
 }
+
+func (s *service) setDatabaseNodes(ctx context.Context) error {
+	var nodeIPList []string
+	nodeList, err := s.pastelClient.MasterNodesExtra(ctx)
+	if err != nil {
+		return err
+	}
+
+	for _, nodeInfo := range nodeList {
+		address := nodeInfo.ExtAddress
+		segments := strings.Split(address, ":")
+		if len(segments) != 2 {
+			return errors.Errorf("malformed db node address: %s", address)
+		}
+		nodeAddress := fmt.Sprintf("%s:%d", segments[0], s.config.RaftPort)
```

Comment:
can you pls make sure we use raftport here and not http port @nguyen202 @danileng20.
related ticket: https://freedcamp.com/view/2956207/tasks/42099740